### DISCUSSION TITLE
Where do logs actually show up?

### DISCUSSION THREAD

**Discussion by Alkamist** [free-audio/clap#322] [Q&A] [answered]
Ok super noob question but generally what are hosts expected to do with logs? I tried logging with Bitwig and Reaper and I can't seem to find where to view them.

**Comment by abique** [comment, marked as answer]
Currently bitwig will put them into the audio engine log file

**Reply by Alkamist** [reply]
Ok so I will need to figure out where that would be for Reaper, but for anyone on Windows using Bitwig it is in C:\Users\YourUserName\AppData\Local\Bitwig Studio\engine.log
